### PR TITLE
Homework 14 done

### PR DESCRIPTION
**Pull request by MaksServatyak**

### CODE REVIEW

**Review by AndrewRorat — APPROVED** [review]

**Review by natalia-kozak — APPROVED** [review]

File: src/Homework14/FIFOQueue.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package Homework14;
+
+import java.util.List;
+
+public class FIFOQueue implements Queue {
+    List<String> queue= new java.util.ArrayList();
```
Comment (Volodymyr95) [inline src/Homework14/FIFOQueue.java:6]: java.util. ?

File: src/Homework14/FIFOQueue.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package Homework14;
+
+import java.util.List;
+
+public class FIFOQueue implements Queue {
```
Comment (Volodymyr95) [inline src/Homework14/FIFOQueue.java:5]: why queue? you have LinkedList with methods which give for you functionality for create

File: src/Homework14/Main.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        List<String> list = new ArrayList();
+        List<String> linkedList = new LinkedList<>();
+
+
+        var list1 = new ArrayList<String>();
+        Homework14.ArrayList arrayList = new Homework14.ArrayList();
```
Comment (Volodymyr95) [inline src/Homework14/Main.java:14]: Incorrect name

File: src/Homework14/Main.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        arrayList.fillList(100000);
```
Comment (Volodymyr95) [inline src/Homework14/Main.java:15]: you should have make metrik how much time

**Pull request merged**